src/package-manager.coffee

```diff
@@ -409,6 +412,18 @@ class PackageManager
     message = "Failed to load the #{path.basename(packagePath)} package"
     atom.notifications.addError(message, {stack, detail, dismissable: true})
 
+  # TODO: remove these autocomplete-plus specific helpers after a few versions.
+  uninstallAutocompletePlus: ->
+    packageDir = null
+    devDir = path.join("dev", "packages")
+    for packageDirPath in @packageDirPaths
+      packageDir = packageDirPath if not packageDirPath.endsWith(devDir)
+
+    if packageDir?
+      autocompletePlusPath = path.join(packageDir, 'autocomplete-plus')
+      fs.isDirectory autocompletePlusPath, (isDir, error) ->
```

Comment:
`fs.isDirectory` calls back with only a single boolean parameter, so `error` here will always be `undefined`.